RELEASE.md

```diff
@@ -11,6 +11,7 @@
 # Upcoming Release 0.18.5
 
 ## Major features and improvements
+* Add the `--conf-source` option to `kedro run`; allowing users to specify a source for project configuration for the run.
```

Comment:
```suggestion
* Add the `--conf-source` option to `kedro run`, allowing users to specify a source for project configuration for the run.
```
More a nitpick than anything else - iirc it'd be more appropriate to use a comma instead of a semicolon here; a good rule of thumb is seeing if replacing the semicolon with a comma and conjunction (and, but, or, e.t.c) would still hold.

Reply:
Changed in [bccb13b](https://github.com/kedro-org/kedro/pull/2117/commits/bccb13baaa0994fcf863dc698dbb98a268756262).